app/components/publications/publication/proofs/proof-requested-panel.hbs

```diff
@@ -0,0 +1,99 @@
+<div class="vlc-procedure-step vlc-procedure-step--nopadding">
+  <div class="vlc-procedure-step__badge-wrapper">
+    <Auk::Badge
+      @icon="mail"
+      @skin="primary"
+      @size="medium"
+      class="vlc-procedure-step__badge"
+    />
+  </div>
+  <Auk::AccordionPanel @isDefaultActive={{true}} as |accordionPanel|>
+    <accordionPanel.Header
+      @size="large"
+      @title={{t "requested-proof"}}
+      @subtitle={{moment-format @requestActivity.startDate "DD.MM.YYYY"}}
+      as |header|
+    >
+      {{#unless this.loadProofingActivity.isRunning}}
+        <header.Actions as |actions|>
+          <actions.Item>
+            {{#unless this.isProofingActivityFinished}}
+              <DropdownMenu @skin="borderless" @icon="three-dots" as |Menu|>
+                <Menu.Item
+                  @skin="danger"
+                  @disabled={{this.isProofingActivityFinished}}
```

Comment:
Aangezien het hele menu niet getoond wordt indien `{{this.isProofingActivityFinished}}` is dit overbodig.